fetch.bs

```diff
@@ -8481,6 +8615,75 @@ with a <var>promise</var>, <var>request</var>, <var>responseObject</var>, and an
 </div>
 
 
+<h3 id=fetch-later-method>FetchLater method</h3>
+
+<pre class=idl>
+
+dictionary DeferredRequestInit : RequestInit {
+  DOMHighResTimeStamp backgroundTimeout;
+};
+
+interface FetchLaterResult {
+  readonly attribute boolean sent;
+};
+
+partial interface mixin WindowOrWorkerGlobalScope {
+  [NewObject] FetchLaterResult fetchLater(RequestInfo input, optional DeferredRequestInit init = {});
```

Comment:
noamr@ any thought on this?

Reply:
- Re `keepalive`, as per current spec it would drop only requests that have already started, meaning requests started due to `backgroundedTimeout`. But yea, makes sense that fetchLater requests are always keepalive.
- Can also be `DELETE` or `PUT`, no? I don't see why we need to limit that because of "the goal of the original pending beacon proposal".

Reply:
> Re `keepalive`

Thanks. Should the API throw exception when `keepalive` is set?

Reply:
@noamr friendly ping

Reply:
I don't think we need to do anything special with keepalive. It's a noop.

Reply:
Is it noop? Should we at least enforce `keepalive=true` in the spec?

Per similar API `navigator.sendBeacon()`'s [spec](https://www.w3.org/TR/beacon/), which explicitly says `The user agent MUST initiate a fetch with keepalive flag set ...`

Reply:
Yes but here that's implied by the algorithm. So no need to pass it from the outside.

Reply:
Resolved